scripts/perftune.py

```diff
@@ -1318,11 +1318,8 @@ def parse_options_file(prog_args):
             raise Exception("Bad 'mode' value in {}: {}".format(prog_args.options_file, y['mode']))
         prog_args.mode = y['mode']
 
-    if 'nic' in y and not prog_args.nic:
-        prog_args.nic = y['nic']
-
-    if 'nics' in y and not prog_args.nics:
-        prog_args.nics.extend(y['nics'])
```

Comment:
dump_config() will only output 'nic' option, and we don't have 'nics' option in the old version.
So parse_options_file() will never see 'nics' option in the options file.

And in current code, prog_args.nic won't be used, only prog_args.nics is used. So the parsed nic value should be extended to prog_args.nics.

I'm ok this above change.